test/wiki-testmap.json

```diff
@@ -41,5 +41,11 @@
         "fedora-wiki-testcase": "QA:Testcase_partitioning_webui_custom_software_RAID",
         "section": "WebUI custom storage configuration",
         "milestone": "Beta"
+    },
+    {
+        "testname": "TestStorageEraseAll_Fedora.testBasic",
+        "fedora-wiki-testcase": "QA:Testcase_partitioning_webui_guided_delete_all",
+        "section": "Guided storage configuration",
+        "milestone": "Basic"
```

Comment:
Maybe not rebooting should be rather approved by Fedora QA?
I completely understand the reason but they are still kind of owners of the test suite?

Reply:
I agree, you are right. I will split this commit to seperate PR and ask Adam to review and discuss it there.